charts/radar-jdbc-connector/values.yaml

```diff
@@ -303,3 +303,25 @@ timescaledb:
       initdb:
         database: database_name
         owner: database_name
+
+    backups:
+      enabled: true
+      endpointURL: http://minio:9000
+
+      provider: s3
+      s3:
+        bucket: "cloudnative-timescaledb"
```

Comment:
What happens when we create three different jdbc-connectors (e.g. _grafana-timescaledb_, _data-dashboard-timescaledb_, ...)? We should not forget to inject new values for this field in the helmfile layer?